File: tys-spa/ciudadano/src/app/domain/remitente.ts
import {Direccion} from './direccion';

export interface Remitente {
  idtipoPersona: number;
  idtratamientoCortesia: number;
  idenCalidadDe: number;
  idtipoDocumentoIdentidad: number;
  idtipoTelefono: number;
  nit: number;
  razonSocial: string;
  numeroDocumento: string;
  nombre: string;
  telefono: string;
  extension: number;
  indicativo: number;
  correoElectronico: string;
  direccion: Direccion;
  codigoActividad: number;
  idTipoEmpresa: number;
  primerNombre: string;
  segundoNombre: string;
  primerApellido: string;
  segundoApellido: string;
}

File: tys-application/tys-commons/src/main/java/co/com/mintrabajo/tys/commons/domain/Remitente.java
package co.com.mintrabajo.tys.commons.domain;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@AllArgsConstructor
@Builder(builderMethodName = "newInstance")
public class Remitente implements Serializable {
	
	private static final long serialVersionUID = 1L;
	
	private Long idtipoPersona;
	private Long idtratamientoCortesia;
	private Long idenCalidadDe;
	private Long idtipoDocumentoIdentidad;
	private Long idtipoTelefono;
	private String nit;
	private String razonSocial;
	private String numeroDocumento;
	private String nombre;
	private Long indicativo;
	private String telefono;
	private Long extension;
	private String correoElectronico;
	private Direccion direccion;

	private String primerNombre;
	private String segundoNombre;
	private String primerApellido;
	private String segundoApellido;

	public Remitente(){
		
	}
	
}
